app/controllers/spree/products_controller.rb

```diff
@@ -107,7 +107,15 @@ def etag_show
     alias product_etag etag_show
 
     def last_modified_index
-      products_last_modified      = @products.maximum(:updated_at)&.utc if @products.respond_to?(:maximum)
+      if @products.respond_to?(:maximum)
```

Comment:
@mrbrdo just wondering - are there any cases where `@products` won't respond to `.maximum`?

Reply:
I don't think so (except if @products is nil but I don't think that's an allowed state). I didn't add that `if`, just kept it from before (was at end of line). And git blame shows it being copied around from another place, I didn't dig deeper.